src/components/BlogSection.tsx

```diff
@@ -109,7 +111,7 @@ const BlogSection = ({
           <span className="publisher-name">{fullName}</span>
           <ShareIcon onClick={() => copyToClipboard(id)} />
         </UtilContainer>
-        <p className="description">{description}</p>
+        <ReactMarkdown className="description" remarkPlugins={[remarkGfm]}>{description}</ReactMarkdown>
```

Comment:
I think we can go with the route that allows us to render both normal content as well markdown content based on the `type` of blog that's coming from the backend(here supabase). We'll need to introduce that type. Here we can just conditionals to check the type of the blog object. For testing and introducing the new column I'll make the changes soon.

column will be `body_content_format`: `TEXT` or `Markdown`.